frontend/ui/widget/inputtext.lua

```diff
@@ -233,17 +233,18 @@ local function initTouchEvents()
                         },
                         {
                             {
-                                text = _("Cancel"),
+                                text = _("Delete all"),
                                 callback = function()
                                     UIManager:close(clipboard_dialog)
+                                    self:delAll()
                                 end,
                             },
                             {
                                 text = _("Select"),
                                 callback = function()
                                     UIManager:close(clipboard_dialog)
                                     UIManager:show(Notification:new{
-                                        text = _("Set cursor to start of selection, then hold."),
+                                        text = _("Set cursor to start of selection, then long-press in text box."),
```

Comment:
Not too long for a single-line notification?
I dunno what happens when too long: truncated ? overflow ? which side gets to be not visible?

Reply:
On your device with your Screen dpi settings and the short English language, and probably for most users, it feels ok.
Just curious about my other questions and if it will reads right if truncated.

Reply:
Ok, missed your 2nd screenshot. Feels alright then.